pkg/output/table.go

```diff
@@ -94,12 +94,22 @@ func ToTable(resourceCostBreakdowns []costs.ResourceCostBreakdown) ([]byte, erro
 			totalHourly = totalHourly.Add(priceComponentCost.HourlyCost)
 			totalMonthly = totalMonthly.Add(priceComponentCost.MonthlyCost)
 
+			hourlyCost := formatDecimal(priceComponentCost.HourlyCost, "%.4f")
```

Comment:
should we apply a similar logic to the json.go file?

Reply:
that's up to you guys, do you want the JSON output to include a placeholder label @alikhajeh1 ?

Reply:
We can add that when we update the json output to include more details.